flow/flow.go

```diff
@@ -894,6 +899,123 @@ func (f *Flow) newTransportLayer(packet *Packet, opts Opts) error {
 
 	return nil
 }
+func newDNSRecordFromLayersRecord(rr *layers.DNSResourceRecord) fl.DNSResourceRecord {
+	return fl.DNSResourceRecord{
+		Name:       string(rr.Name),
+		Type:       rr.Type.String(),
+		Class:      rr.Class.String(),
+		TTL:        rr.TTL,
+		DataLength: rr.DataLength,
+	}
+}
+func toFlowDNSResourceRecord(rr *layers.DNSResourceRecord) (fl.DNSResourceRecord, error) {
+	switch rr.Type {
+	case layers.DNSTypeA:
```

Comment:
I guess it should be case layers.DNSTypeA, layers.DNSTypeAAAA ?

Reply:
you're correct. My bad.
Fixed